### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by KaterynaEV**
- [DEMO LINK](https://KaterynaEV.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://KaterynaEV.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,7 +11,32 @@
...
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="main">
+      <form class="search__bar content__form"
+      method="get"
+      data-qa="big">
```
Comment (VitaliyBodnar) [inline src/index.html:20]: code it not aligned 
Screenshot 2023-10-18 at 13 15 10

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+.content__form {
+  margin: 20px 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.search__bar {
+  display: flex;
+  height: 70px;
+  border: 1px solid #e1e7ed;
```
Comment (VitaliyBodnar) [inline src/style.css:28]: It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+.main {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```
Comment (VitaliyBodnar) [inline src/style.css:16]: box-sizing: border-box; is duplicated, so make sense to move it to the `*{ box-sizing: border-box }`

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

**Pull request closed without merging**